tests/client/test_workspaces.py

```diff
@@ -177,3 +177,43 @@ def test_init_with_missing_workspace(owner: "ServerUser"):
 
     with pytest.raises(ValueError):
         rg.init(api_key=owner.api_key, workspace="missing-workspace")
+
+
+@pytest.mark.asyncio
+async def test_delete_workspace(owner: "ServerUser"):
```

Comment:
```suggestion
async def test_delete_workspace(owner: "ServerUser") -> None:
```

Reply:
I'm not sure about this if the function is not explicitly returning anything. What are the best practices for defining this?

Reply:
I guess the standard is to `-> None`, to make it more explicit, otherwise the default will be `Any` or the inferred return-type if any.

Reply:
For me, it makes sense to add a return signature in a function definition if the function has an explicit `return` statement or the function will be called somewhere. Normally a test doesn't need to return anything and shouldn't be called in other contexts. So adding the `-> None` signature to a test is not necessary.